documentation.go

```diff
@@ -417,6 +417,9 @@ func (d *documentationCommand) formatFlags(c Command, info *Info) string {
 	f.VisitAll(func(f *gnuflag.Flag) {
 		flags[f.Value] = append(flags[f.Value], f)
 	})
+	if len(flags) == 0 {
+		return ""
```

Comment:
It would be nice to pass something else back to indicate nothing to do, but for now, this makes a lot of sense.